avogadro/io/xyzformat.cpp

```diff
@@ -52,12 +53,28 @@ std::optional<double> findEnergy(const std::string& buffer)
     energyStart = buffer.find(" E ");
     offset = 3;
   }
+  if (energyStart == std::string::npos) {
+    energyStart = buffer.find("Epot = ");
+    offset = 6;
+  }
 
   if (energyStart != std::string::npos) {
     // pick the next token
     std::string energy = buffer.substr(energyStart + offset);
     return lexicalCast<double>(energy);
   }
+
+  // also check if the comment just starts with a number
+  // try to split out the first token and look for a number
+  std::vector<string> tokens(split(buffer, ' '));
+  if (!tokens.empty()) {
+    bool ok(false);
+    auto energy = lexicalCast<double>(tokens[0], ok);
+    if (ok)
+      return energy;
+    // otherwise we fall through to return std::nullopt
```

Comment:
can be simplified into
```suggestion
    return lexicalCast<double>(tokens[0]);
```